code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -314,6 +314,71 @@
 	l_pocket = /obj/item/tank/internals/emergency_oxygen
 	r_pocket = /obj/item/flashlight/glowstick
 
+//Moffuchi's pizza parlor,located in the icemoon's caves, is home to chef Moffuchi and his apprentice. They run their resturant for hungery miners and adventurers, but mostly for cash
+/obj/effect/mob_spawn/human/mothchef
+	name = "locked freezer"
+	desc = "You can hear fluttering from inside this freezer,<i>is there a moth in there</i>?"
+	mob_name = "pizzaria manager"
+	icon = 'icons/obj/crates.dmi'
+	icon_state = "freezer"
+	roundstart = FALSE
+	death = FALSE
+	random = TRUE
+	id_job = "Pizzaria Manager"
+	id_access = "cook"
+	mob_species = /datum/species/moth
+	short_desc = "You are the moon-renowned chef Moffuchi."
+	flavour_text = "Now that you have escaped the freezer you locked yourself in last night, it is time to bring glory \
+	to the Moffuchi name by making pizzas for all to enjoy! Though the nearby and newly constructed Nanotrasen research facility \
+	has its own inferior cafeteria, optimism leads you to belive the new arrivals may become regulars at the pizza parlor!"
+	uniform = /obj/item/clothing/under/rank/civilian/chef
+	shoes = /obj/item/clothing/shoes/sneakers/black
+	mask = /obj/item/clothing/mask/fakemoustache/italian
+	suit = /obj/item/clothing/suit/toggle/chef
+	l_pocket = /obj/item/seeds/tomato
+	id = /obj/item/card/id
+	id_access_list = list(28)
+
+/obj/effect/mob_spawn/human/mothchef/Destroy()
+	new/obj/structure/closet/crate/freezer(get_turf(src))
+	return ..()
+
+/obj/effect/mob_spawn/human/mothchef/special(mob/living/L)
+	//Possible names based on famous chefs, real or fictional.
+	var/moth_name = pick("Hank","Gordon","Guy","Chef","Willy","Brad","Bobby","Alton")
+	L.fully_replace_character_name(null,"[moth_name] Moffuchi")
+	var/mob/living/carbon/human/H = L
+	var/obj/item/worn = H.wear_id
+	var/obj/item/card/id/id = worn.GetID()
+	id.registered_name = L.real_name
+	id.update_label()
+
+/obj/effect/mob_spawn/human/mothsouschef
+	name = "locked freezer"
+	desc = "You can hear fluttering from inside this freezer,<i>is there a moth in there</i>?"
+	mob_name = "pizzaria assistant manager"
+	icon = 'icons/obj/crates.dmi'
+	icon_state = "freezer"
+	roundstart = FALSE
+	death = FALSE
+	random = TRUE
+	id_job = "Pizzaria Assistant Manager"
```

Comment:
```suggestion
	id_job = "Pizzeria Assistant Manager"
```

Reply:
I remember finding out i was spelling it wrong early on, then I told myself I would find and replace them with the correct spelling. I guess i forgot to do that. Thanks for catching that.